editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/Wsdl2RestWizard.java

```diff
@@ -0,0 +1,294 @@
+		IResource resource = ResourcesPlugin.getWorkspace().getRoot().findMember(javaPath);
+		if (resource == null) {
+			IFolder folder = project.getFolder(javaPath.removeFirstSegments(1));
+			if (!folder.exists()) {
+				prepare(folder);
+				resource = folder;
+				project.refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
+			}
+		}
+
+		// use project to determine if we are building a spring or blueprint project
+		boolean isBlueprint = isProjectBlueprint(project);
+		
+		File javaFile = null;
+		if (resource instanceof IFolder) {
+			IFolder destFolder = (IFolder) resource;
+			// gets URI for EFS.
+			URI uri = destFolder.getLocationURI();
+
+			// what if file is a link, resolve it.
+			if(destFolder.isLinked()){
+				uri = destFolder.getRawLocationURI();
+			}
+
+			// Gets native File using EFS
+			javaFile = EFS.getStore(uri).toLocalFile(0, new NullProgressMonitor());			
+		}
+		IPath camelPath = new org.eclipse.core.runtime.Path(options.getDestinationCamel());
+		IResource camelResource = ResourcesPlugin.getWorkspace().getRoot().findMember(camelPath);
+		File camelFile = null;
+		if (camelResource instanceof IFolder) {
+			IFolder camelDestFolder = (IFolder) camelResource;
+			// gets URI for EFS.
+			URI cameluri = camelDestFolder.getLocationURI();
+
+			// what if file is a link, resolve it.
+			if(camelDestFolder.isLinked()){
+				cameluri = camelDestFolder.getRawLocationURI();
+			}
+
+			// Gets native File using EFS
+			camelFile = EFS.getStore(cameluri).toLocalFile(0, new NullProgressMonitor());			
+		}
+		if (javaFile != null) {
+			ClassLoader oldLoader = Thread.currentThread().getContextClassLoader();
+			try {
+				// initialize bus using bundle classloader, to prevent project dependencies from leaking in
+				Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
+
+				Path outJavaPath = javaFile.toPath();
+				Wsdl2Rest tool = new Wsdl2Rest(wsdlLocation, outJavaPath);
+				if (!isBlueprint) {
+					if (camelFile != null) {
+						Path contextpath = new File(camelFile.getAbsolutePath() + File.separator + "rest-camel-context.xml").toPath(); //$NON-NLS-1$
+						tool.setCamelContext(contextpath); 
+					} else {
+						IPath projectPath = ResourcesPlugin.getWorkspace().getRoot().findMember(project.getName()).getLocation();
+						Path contextpath = new File(projectPath.makeAbsolute() + File.separator + "rest-camel-context.xml").toPath(); //$NON-NLS-1$
+						tool.setCamelContext(contextpath); 
+					}
+				} else {
+					if (camelFile != null) {
+						Path contextpath = new File(camelFile.getAbsolutePath() + File.separator + "rest-blueprint-context.xml").toPath(); //$NON-NLS-1$
+						tool.setCamelContext(contextpath); 
+					} else {
+						IPath projectPath = ResourcesPlugin.getWorkspace().getRoot().findMember(project.getName()).getLocation();
+						Path contextpath = new File(projectPath.makeAbsolute() + File.separator + "rest-blueprint-context.xml").toPath(); //$NON-NLS-1$
```

Comment:
this variable is not used. Is it normal? If normal, the whole line can be removed

Reply:
Fixed